main.go

```diff
@@ -438,9 +450,18 @@ func touchSerialPortAt1200bps(port string) error {
 
 func flashUF2UsingMSD(volume, tmppath string) error {
 	// find standard UF2 info path
-	infoPath := "/media/*/" + volume + "/INFO_UF2.TXT"
-	if runtime.GOOS == "darwin" {
+	var infoPath string
+	switch runtime.GOOS {
+	case "linux":
+		infoPath = "/media/*/" + volume + "/INFO_UF2.TXT"
+	case "darwin":
 		infoPath = "/Volumes/" + volume + "/INFO_UF2.TXT"
+	case "windows":
+		path, err := windowsFindUSBDrive()
+		if err != nil {
+			return err
+		}
+		infoPath = path + "/INFO_UF2.TXT"
```

Comment:
Looks like this only checks the first FAT device available (that may or may not be the right one).
That can possibly be fixed later, but could be surprising to new users if no device can be found while it's clearly plugged in (along a USB thumb drive).

Reply:
Added some additional logic to ensure that we only use attached volume with the correct name that matches the board, like we are doing on Linux/macOS.

Reply:
I don't see the additional logic, but I do see that I missed the extra `volume` parameter to `windowsFindUSBDrive` which makes sure that only a volume with the correct name is returned. So that should be good.